Gopkg.toml

```diff
@@ -92,3 +88,7 @@ required = ["github.com/golang/protobuf/jsonpb"]
 [[constraint]]
   name = "github.com/opentracing/basictracer-go"
   version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
can we change `branch` to `version` to make tidb not depend on the `master` branch of `gofail`?

Reply:
In case they change the path again in the future.

Reply:
gofail change the path, ci failed not caused by the version.

Reply:
Master is the only branch of go-fail. And they do not have any release.